microprot/scripts/split_search.py

```diff
@@ -529,19 +574,21 @@ def mask_sequence(hhsuite_fp, fullsequence_fp, subsequences_fp=None,
 
     for hit in subseqs_pos:
         _id = get_q_id(hit)
-        header = "%s_%i-%i %s" % (query_id,
-                                  hit['alignment'][_id]['start'],
-                                  hit['alignment'][_id]['end'], query_desc)
+        header = "%s %s" % (correct_header_positions(
+            query_id,
+            hit['alignment'][_id]['start'],
+            hit['alignment'][_id]['end']), query_desc)
```

Comment:
True - definitely not in the scope of this PR.  But I wouldn't let that halt future refactoring efforts.  If we want, we can always gut the scikit-bio sequence object and tailor it for our own use cases -- since it is a nice useful object.  And this can be beneficial for scikit-bio in the future, since this will motivate additional useful components that can be added in.